javascript-loops-practice/filter-out-strings.js

```diff
@@ -0,0 +1,9 @@
+/* exported filterOutStrings */
+function filterOutStrings(values) {
+  var noStrings = [];
+  for (let i = 0; i < values.length; i++) {
+    if (typeof values[i] === 'number') {
```

Comment:
This looks like it is collecting all numbers, not excluding strings 🤔 . What if the input array included null or undefined or booleans etc?